test/functional/mempool_accept.py

```diff
@@ -42,7 +42,7 @@ class MempoolAcceptanceTest(BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 1
         self.extra_args = [[
-            '-txindex','-permitbaremultisig=0',
+            '-txindex','-permitbaremultisig=0', '-mempoolfullrbf=0'
```

Comment:
Seems easier and clearer to just fixup the test?


```diff
diff --git a/test/functional/mempool_accept.py b/test/functional/mempool_accept.py
index 02ec18140c..be9cea95c1 100755
--- a/test/functional/mempool_accept.py
+++ b/test/functional/mempool_accept.py
@@ -140,7 +140,14 @@ class MempoolAcceptanceTest(BitcoinTestFramework):
         tx = tx_from_hex(raw_tx_0)
         tx.vout[0].nValue -= int(4 * fee * COIN)  # Set more fee
         self.check_mempool_result(
-            result_expected=[{'txid': tx.rehash(), 'allowed': False, 'reject-reason': 'txn-mempool-conflict'}],
+            result_expected=[{
+                'txid': tx.rehash(),
+                'allowed': True,
+                'vsize': 104,
+                'fees': {
+                    'base': Decimal('0.00004200')
+                },
+            }],
             rawtxs=[tx.serialize().hex()],
             maxfeerate=0,
         )
diff --git a/test/functional/p2p_permissions.py b/test/functional/p2p_permissions.py
index 453a0920cc..52caceee24 100755
--- a/test/functional/p2p_permissions.py
+++ b/test/functional/p2p_permissions.py
@@ -112,8 +112,7 @@ class P2PPermissionsTests(BitcoinTestFramework):
                 }], outputs=[{
                     ADDRESS_BCRT1_P2WSH_OP_TRUE: 5,
                 }],
-                replaceable=False),
-        )
+        ))
         tx.wit.vtxinwit = [CTxInWitness()]
         tx.wit.vtxinwit[0].scriptWitness.stack = [CScript([OP_TRUE])]
         txid = tx.rehash()
@@ -130,13 +129,13 @@ class P2PPermissionsTests(BitcoinTestFramework):
         self.log.debug("Check that node[1] will not send an invalid tx to node[0]")
         tx.vout[0].nValue += 1
         txid = tx.rehash()
-        # Send the transaction twice. The first time, it'll be rejected by ATMP because it conflicts
-        # with a mempool transaction. The second time, it'll be in the m_recent_rejects filter.
+        # Send the transaction twice. The first time, it'll be rejected by the
+        # mempool. The second time, it'll be in the m_recent_rejects filter.
         p2p_rebroadcast_wallet.send_txs_and_test(
             [tx],
             self.nodes[1],
             success=False,
-            reject_reason='{} from peer=0 was not accepted: txn-mempool-conflict'.format(txid)
+            reject_reason=f"{txid} from peer=0 was not accepted: insufficient fee",
         )
 
         p2p_rebroadcast_wallet.send_txs_and_test(

Reply:
I thought it was better to keep testing opt-in rbf conflict handling given we've still got the opt-in rbf code. Doing minimal changes to the tests maybe helps show this is a defaults change, not a behaviour change, too? No strong opinion.

Reply:
I think test/functional/feature_rbf.py does the opt-in checks? The other tests shouldn't really deal with rbf testing at all. Though, it is just a nit.